Jenkinsfile

```diff
@@ -5,6 +5,9 @@ library("govuk")
 node("postgresql-9.6") {
```

Comment:
Should this node (name?) be updated too?

Reply:
i'm just checking that...I wonder if we can remove the argument altogether as i've not seen it in other apps...

Reply:
Yeah, maybe. I've just realised it's also present in Content Data Admin (https://github.com/alphagov/content-data-admin/blob/main/Jenkinsfile#L5).

Reply:
I've removed it from my PR (https://github.com/alphagov/content-data-admin/pull/993) - lets see if the build passes.

Reply:
Build passed - https://ci.integration.publishing.service.gov.uk/job/content-data-admin/job/upgrade-to-postgres-13/3/

Reply:
I've found renaming it fails - the name is not arbitrary. Removing it entirely passes so I assume that is the best route to take at the moment